src/js/program/ad-program-controller.js

```diff
@@ -18,7 +18,7 @@ export default class AdProgramController extends ProgramController {
         // (i.e. prerolls, midrolls, and postrolls must use the same tag)
         let mediaElement;
         if (this.backgroundLoading) {
-            // The media pool has reserves an element for ads to use. It is reserved on setup and is not used by other media
+            // The media pool has reserved an element for ads to use. It is reserved on setup and is not used by other media
```

Comment:
it might be `The media pool reserves an element for ads to use`?

Reply:
You're right. I don't like that comment so I removed it. Read the source.